sdk/python/ragflow_sdk/modules/session.py

```diff
@@ -79,12 +113,14 @@ def ask(self, question="", stream=False, **kwargs):
             yield self._structure_answer(json_data["data"])
         
 
-    def _structure_answer(self, json_data):
+    def _structure_answer(self, json_data) -> "Message":
         if self.__session_type == "agent":
            answer = json_data["data"]["content"]
         elif self.__session_type == "chat":
             answer = json_data["answer"]
-        reference = json_data.get("reference", {})
+        else:
+            raise Exception(f"Unknown session type: {self.__session_type}")
```

Comment:
Noticed a missing `elif` branch and fixed it while I was at it. I mentioned that here:
https://github.com/infiniflow/ragflow/commit/95fad5d5234f69bc4c850413be1b3cba130b0ce3#r170922254